README.md

```diff
@@ -19,3 +19,14 @@ the following features:
     - Pieces of text can be addressed specifically by underlining this text
     - Comments in float enviroments (e.g., figure)
 
+  - Fonts
+    - The [ACMart fonts](https://github.com/opencompl/paper-template/blob/master/acmart.cls#L720-L728)
+      use [Inconsolata](https://ctan.org/pkg/inconsolata?lang=en) for monospace, 
+      [Libertine](https://ctan.org/pkg/libertine?lang=en) for serif, and [newtx](https://ctan.org/pkg/newtx?lang=en) for math.
+    - The Libertine `ttf` can be obtained as "linux libertine" on distros, and Inconsolata `ttf` as "Inconsolata".
+    - The default font size of the body is `11pt`.
+  - Figures
+    - The paper is of A4 size, which in *physical units* is `210 x 297 mm`.
```

Comment:
The ACM journal paper is not of A4 size, but of US letter size.

Reply:
Fixed.